packages/web-twig/README.md

```diff
@@ -85,6 +85,31 @@ considered carefully, and extensively tested. In general, customizing Spirit des
 do offer control over layout and other aspects. In addition, you can use Spirit defined design tokens to ensure your
 application conforms to your design requirements, and is adaptive across platform scales and color schemes.
 
+### Style props
+
+All Spirit components accept a set of props that can be used to control their outer spacing. The props are:
+
+- `margin`
+- `marginTop`
+- `marginRight`
+- `marginBottom`
+- `marginLeft`
+- `marginX`
+- `marginY`
+
+These props accept a spacing token (eg. `space-100`), `auto` or an object with breakpoint keys and spacing token
```

Comment:
Maybe add a full example of the breakpoint keys. I think this will be repeated on multiple places.